internal/controllers/cryostat_controller.go

```diff
@@ -659,6 +676,66 @@ func (r *CryostatReconciler) deleteConsoleLink(ctx context.Context, cr *operator
 	return nil
 }
 
+func (r *CryostatReconciler) addCorsAllowedOriginIfNotPresent(allowedOrigin string, cr *operatorv1beta1.Cryostat) error {
+	reqLogger := r.Log.WithValues("Request.Namespace", cr.Namespace, "Request.Name", cr.Name)
+	apiServer := &configv1.APIServer{}
+	err := r.Client.Get(context.Background(), types.NamespacedName{Name: "cluster"}, apiServer)
```

Comment:
Maybe we should extract the "cluster" string into a constant for these two methods. It might be handy in case this name changes in the future.

Reply:
I stored the constant in `resource_definitions`. Let me know if you think of a better place for it.

For future reference, the "cluster" string name is mentioned in the config.openshift.io docs here: https://docs.openshift.com/container-platform/4.6/rest_api/config_apis/apiserver-config-openshift-io-v1.html

Reply:
I would just put the constant in `cryostat_controller.go` and keep it private visibility (starts with lower case). It's not needed elsewhere I think?

Reply:
Sounds good